server/athenian/api/controllers/miners/github/pull_request.py

```diff
@@ -36,10 +57,28 @@ def __init__(self, prs: pd.DataFrame, reviews: pd.DataFrame, review_comments: pd
         :param repositories: PRs must belong to these repositories (prefix excluded).
         :param developers: PRs must be authored by these user IDs. An empty list means everybody.
         :param db: Metadata db instance.
+        :param cache: memcached client to cache the collected data.
         """
+        cache_key = None
+        if cache is not None:
+            cache_key = gen_cache_key(
+                "PullRequestMiner|%d|%d|%s|%s",
+                time_from.toordinal(),
+                time_to.toordinal(),
+                ",".join(sorted(repositories)),
+                ",".join(sorted(developers)),
+            )
+            serialized = await cache.get(cache_key)
+            if serialized is not None:
+                prs, reviews, review_comments, comments, commits = \
+                    cls._deserialize_from_cache(serialized)
+                return cls(prs, reviews, review_comments, comments, commits)
         filters = [
-            PullRequest.created_at >= time_from,
-            PullRequest.created_at < time_to,
+            sql.or_(sql.and_(PullRequest.updated_at >= time_from,
+                             PullRequest.updated_at < time_to),
```

Comment:
No check needed for `update_at` `None`?

Reply:
Yes, it is guaranteed to be not null - it is at least equal to created_at